server/game/core/ability/CardAbility.ts

```diff
@@ -113,12 +113,6 @@ export class CardAbility extends CardAbilityStep {
                 .map((effect) => effect(context));
```

Comment:
Does anything break if we just removed this method now? I think the other stuff it doesn't is legacy l5r rules that we don't have in swu, so if we can remove it that would increase the clarity of the code overall. Sorry, I meant to ask for you to investigate that as part of cleaning up the parameters of `getCosts` in the other class but I wasn't clear